### PR TITLE
Patch minor issues

### PR DESCRIPTION
**Pull request by fbau3r**
Just because I stumbled upon them, I fixed some typos and removed jquery because it isn't used anywhere.

To be honest, I just want to try out creating a pull request... :blush:

### CODE REVIEW

File: _includes/icon-links.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- annnd guess where these are defined? Yup, you guessed it: the _config.yml file -->
+<!-- and guess where these are defined? Yup, you guessed it: the _config.yml file -->
```
Comment (jglovier) [inline _includes/icon-links.html:1]: the multiple n's were used intentionally (for dramatic effect), but that's okay; they can stay out

File: _layouts/resume.html [hunk 2]
```diff
@@ -199,7 +199,7 @@ <h4 class="resume-item-details">Member in Good Standin &bull; 1994 &mdash; Prese
         </div>
 
       </section>
-      <!-- end Recognition -->
+      <!-- end Associations -->
```
Comment (jglovier) [inline _layouts/resume.html:202]: Ah - good catch!

**Comment by jglovier** [comment]
@fbau3r awesome!! Thanks for the pull, and good on you for stepping into the pull request waters!

:heart: :zap: :metal:

**Comment by fbau3r** [comment]
Thanks for merging during your vacation! :wink:

**Pull request merged**